proxysql/test-proxysql

```diff
@@ -119,7 +119,24 @@ elif [ -f /usr/bin/apt ]; then
 fi
 
 pushd ${WORKDIR_ABS}/WORKDIR
-sudo tar -xzf $WORKDIR_ABS/proxysql*glibc*.tar.gz -C ${WORKDIR_ABS}/PROXY_ADMIN
+unset PROXY_TAR_FILENAME
+
+for X in $(seq 0 60); do
```

Comment:
Use a variable for timeout. Something like `COPYY_TIMEOUT=60`

Reply:
Fixed